code_regen/README.md

```diff
@@ -0,0 +1,84 @@
+goa Code Regeneration
+=====================
+
+Note: This depends on a
+[PR in progress](https://github.com/goadesign/goa/pull/593) for changing TDB to the tag pairs.
+
+---
+
+This example shows how to regenerate your apps code
```

Comment:
might be good to specify a bit more about the context here so people don't think this is required to update the design. This script is just a helper to re-generate a controller so you can quickly take advantage of the scaffold generated for its handlers. at the end of the day the controller code is user code and should be treated as such.